### PR TITLE
TINY-4210: Add toolbar_location documentation

### PR DESCRIPTION
**Pull request by SimonFc**
Documentation for new `toolbar_location` option, for 5.2

### CODE REVIEW

File: _includes/configuration/toolbar-location.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## toolbar_location
+
+The `toolbar_location` option is used to position the [toolbar]({{site.baseurl}}/configure/editor-appearance/#toolbar) and [menubar]({{site.baseurl}}/configure/editor-appearance/#menubar). The default behavior is to position them above the content area but by settings this option to `bottom`, the toolbar and menubar will be positioned below it. This option works in both Iframe and inline mode.
```
Comment (lnewson) [inline _includes/configuration/toolbar-location.md]: `s/by settings/by setting/`?

Comment (lnewson) [reply _includes/configuration/toolbar-location.md]: `s/below it/below the content area/`? As it wasn't quite clear here I thought.

File: _includes/configuration/toolbar-location.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+**Type:** `String`
+
+**Default Value:** `top`
```
Comment (lnewson) [inline _includes/configuration/toolbar-location.md]: Should we add `Possible values` here? eg:
```
**Possible values:** `top` or `bottom`
```

**Comment by tylerkelly13** [comment]
Fixed merge conflicts and GitHub weirdness

**Pull request merged**